docs/guides/account-user-mgmt/index.md

```diff
@@ -0,0 +1,43 @@
+---
+valeOff: <!-- vale off -->
+heading: Manage Accounts and Users
+seo: Account Overview | Cloud Elements  Docs
+title: Overview
+description: Manage organizations, users, and accounts
+layout: sidebarleft
+apis: API Docs
+platform: accounts
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 1
+sitemap: false
+ValeOn: <!-- vale on -->
+---
+
+# Overview
+
+{% include callout.html content="The <i>Manage Accounts and Users</i> guide includes steps to use Cloud Elements 2.0. Earlier versions of Cloud Elements share much of the same functionality. You can read information about the earlier version at <a href=../../platform-api/accounts/account-management.html>Account Usage and Management Guide</a>." type="info" %}
+
+The first user to sign up for Cloud Elements created an organization and established an Organization Secret automatically. As the default user of the organization, they are the organization administrator. The organization administrator can manage the accounts within the organization and the users associated with those accounts.
+
+Your organization comes with a default account and the organization administrator is the first user associated with that account. The organization can include many accounts. Different accounts cannot see information associated with other accounts. Each account includes individual users.  Cloud Elements assigns each user a User Secret. With the Organization Secret representing the company and the User Secret representing a discrete customer of the company, users can make requests to the Cloud Elements APIs.
```

Comment:
I'm not sure if I would use the word `company` here?  Might just be a personal problem since we're always mixing up organization <> company so feel free to disagree here 😄